RealEstate/apps/core/templates/core/realtorHome.html

```diff
@@ -1,5 +1,106 @@
 {% extends "core/base.html" %}
+{% load bootstrap3 %}
+
+{% block style %}
+<style>
+#heading { padding-bottom: 20px; }
+#mail-icon:hover { color: #337ab7;}
+a.mail { text-decoration: none; }
+ul.list-group { padding-top:30px; }
+.report-container { min-width: 300px; }
+.couple-container { min-width: 400px; }
+.red-alert { color: #e74c3c; }
+.vcenter {
+	display: inline-block;
+	vertical-align: middle;
+	float: none;
+	margin-left:-3px;
+}
+#mail-icon {
+    font-size: 90%;
+    padding-left: 20px;
+    color: #cccccc;
+    display: none;
+}
+</style>
+{% endblock %}
 
 {% block body %}
-  <p>Home page for realtors.</p>
+    <div id="heading">
+        <h2><strong>Realtor Dashboard</strong> </h2>
+    </div>
+
+    <ul class="list-group">
+        {% for couple, homebuyers, isPending in couples %}
+        {% if isPending %}
+            <li id="row_p{{couple.id}}" class="list-group-item couple_row">
+        {% else %}
+            <li id="row_{{couple.id}}" class="list-group-item couple_row">
+        {% endif %}
+			<div class="row">
+				<div class="col-sm-7 vcenter couple-container">
+				    {% for homebuyer in homebuyers %}
+				        <h4>
+				            {% if isPending%}
+				                {{homebuyer.email}}
+				                {% if not homebuyer.registered %}<span class="red-alert">*</span> {% endif %}
+				                <a href="mailto:{{homebuyer.email}}" target="_top" class="mail"><span id="mail-icon" class="glyphicon glyphicon-envelope vcenter mail_p{{couple.id}}" ></span></a>
+				            {% else %}
+				                {{homebuyer}}<a href="mailto:{{homebuyer.email}}" target="_top" class="mail"><span id="mail-icon" class="glyphicon glyphicon-envelope vcenter mail_{{couple.id}}" ></span></a>
+				            {% endif %}
+				        </h4>
+	                {% endfor %}
+				</div>
+				<div class="col-sm-5 vcenter report-container">
+				    <center>
+				        <a class="report" href="{{couple.report_url}}">Report</a>
+				    </center>
+				</div>
+			</div>
+		</li>
+		{% endfor %}
+    </ul>
+    {% if hasPending %}
+        <div class="red-alert">* Pending homebuyers</div>
+    {% endif %}
+    <center>
```

Comment:
Google uses "center" only for its homepage, mainly because the user will see the page only once, which means caching will not play a great role here.  But if you have a lot of javascript and CSS in the page, caching should be considered to speed up the loading process. Why using all HTML 5 except this center tag to diminish this power?